[2025-08-07 03:21:14,874] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_task_group.section_2.inner_section_2.task_2 scheduled__2025-08-06T03:21:08.906347+00:00 [queued]>
[2025-08-07 03:21:14,907] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_task_group.section_2.inner_section_2.task_2 scheduled__2025-08-06T03:21:08.906347+00:00 [queued]>
[2025-08-07 03:21:14,910] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 03:21:14,911] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-08-07 03:21:14,912] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 03:21:14,952] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): section_2.inner_section_2.task_2> on 2025-08-06 03:21:08.906347+00:00
[2025-08-07 03:21:14,973] {standard_task_runner.py:52} INFO - Started process 387 to run task
[2025-08-07 03:21:14,982] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_task_group', 'section_2.inner_section_2.task_2', 'scheduled__2025-08-06T03:21:08.906347+00:00', '--job-id', '22', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_task_group.py', '--cfg-path', '/tmp/tmpyh12bl4l', '--error-file', '/tmp/tmprghdkpsa']
[2025-08-07 03:21:14,984] {standard_task_runner.py:80} INFO - Job 22: Subtask section_2.inner_section_2.task_2
[2025-08-07 03:21:15,131] {task_command.py:369} INFO - Running <TaskInstance: example_task_group.section_2.inner_section_2.task_2 scheduled__2025-08-06T03:21:08.906347+00:00 [running]> on host f83ee74b89ef
[2025-08-07 03:21:15,708] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_task_group
AIRFLOW_CTX_TASK_ID=section_2.inner_section_2.task_2
AIRFLOW_CTX_EXECUTION_DATE=2025-08-06T03:21:08.906347+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-06T03:21:08.906347+00:00
[2025-08-07 03:21:15,709] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-07 03:21:15,710] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2025-08-07 03:21:15,728] {subprocess.py:85} INFO - Output:
[2025-08-07 03:21:15,729] {subprocess.py:92} INFO - 1
[2025-08-07 03:21:15,730] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-07 03:21:15,921] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_task_group, task_id=section_2.inner_section_2.task_2, execution_date=20250806T032108, start_date=20250807T032114, end_date=20250807T032115
[2025-08-07 03:21:16,036] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-07 03:21:16,172] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
